src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -404,6 +454,13 @@ Env::meta()
     return item.second;
 }
 
+std::shared_ptr<STTx const>
+Env::tx()
+{
+    auto const item = current()->txRead(txid_);
+    return item.first;
+}
```

Comment:
I don't think RVO will kick in, but implicit move will kick in.  This would be fine.